.github/workflows/ok-to-test.yml

```diff
@@ -0,0 +1,18 @@
+name: Merge the current PR
```

Comment:
```suggestion
name: Add the ok-to-test label
```

You should update all the other names in this workflow, otherwise it becomes a mess